src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const renderAge = () => {
+    if (person.age) {
+      return <p className="Person__age">I am {person.age}</p>;
```

Comment:
Destructuring can be used. This makes the code cleaner and more user-friendly
```suggestion
export const Person = ({ person }) => {
  const { name, age, sex, isMarried, partnerName } = person;
  
  const renderAge = () => {
    if (age) {
      return <p className="Person__age">I am {age}</p>;
```